types/request_type.go

```diff
@@ -139,6 +140,14 @@ func (r *Request) Sign(signer *asymmetric.PrivateKey) (err error) {
 	return r.Header.Sign(signer)
 }
 
+func (r *Request) SetMarshalCache(buf []byte) {
+	r._marshalCache = buf
+}
+
+func (r *Request) GetMarshalCache() (buf []byte) {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method Request.GetMarshalCache should have comment or be unexported